src/components/layout/side-menu/SideMenu.js

```diff
@@ -0,0 +1,57 @@
+import { AppBar } from '@material-ui/core';
+import {  
+  Home,  
+  Devices,
+  Sensors,
+  History,
+  LocalHospital,
```

Comment:
```suggestion
  LocalHospital,
```
Remover import que não esta sendo usado.